woocommerce/woocommerce-android
# Analytics hub tracks

Summary
==========
Fix issue #7588 by introducing a set of tracks to the Analytics Hub group of features.

Expect the following tracks to be introduced here:

`_dashboard_see_more_analytics_tapped`: Tapping on “See more” in the My Store screen.
`_analytics_hub_date_range_button_tapped`: Tapping on the date range selector button.
`_analytics_hub_date_range_option_selected`: 
    - Custom properties: `option`: The option selected like “today”, “yesterday”, or “custom”. Make sure that this is not translated into the local language.
`_analytics_hub_see_report_tapped`: Tapping the “See report” button.
    - Custom properties:`card`: The card type in English, e.g., “revenue”, “orders”.

How to Test
==========
1. Open the Woo app with the Analytics Hub feature flag enabled
2. Click on the `See more` button inside the My Store stats
3. Verify that the Analytics Hub opens, click the date selector on the top right corner and change it
4. Click on the `See Report` button
5. Return from the browser back to the Analytics Hub

After doing all this flow, verify inside the logs the presence of the tracks as described in the Summary.

Update release notes:
